src/views/ManageList.jsx

```diff
@@ -55,6 +59,20 @@ export function ManageList() {
 		[itemName, daysUntilNextPurchase, listPath],
 	);
 
+	const handleEmailInputSubmit = async (event) => {
+		event.preventDefault();
+		const listAdded = await shareList(listPath, userId, emailData);
+		if (listAdded === 'match') {
+			alert('You cannot share the list with yourself.');
+		} else if (listAdded === true) {
```

Comment:
I'm wondering if there's an opportunity to set the email data to an empty string after a successful submission. This would automatically clear the field, providing great feedback to the user and signalling that the field is ready for a new email!

Reply:
very fair! added a setter function to set the input to an empty string at the end of the submission handler!